source/blood/src/actor.cpp

```diff
@@ -2915,7 +2904,7 @@ spritetype *actDropObject(spritetype *pSprite, int nType) {
     spritetype *pSprite2 = NULL;
     
 #ifdef NOONE_EXTENSIONS
-    if (!VanillaMode() && IsUserItem(nType))
+    if (/*!VanillaMode() &&*/ IsUserItem(nType))
     {
```

Comment:
Could you remove the three instances of `/*!VanillaMode() &&*/ ` comments? I don't see a reason to keep them.